src/main/java/core/basesyntax/model/User.java

```diff
@@ -58,4 +64,13 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(login, password, age);
```

Comment:
Including the password in the `hashCode` method might not be ideal for security reasons. Consider whether it's necessary to include passwords in hash calculations.